src/os/bluestore/bluefs_types.h

```diff
@@ -88,17 +95,16 @@ struct bluefs_fnode_t {
     auto it = extents.begin();
     allocated -= it->length;
     extents.erase(it);
+    extents_index.erase(extents_index.begin());
```

Comment:
wouldn't this also adjust all of the extents_index values?

Reply:
fixed